packages/wonder-blocks-data/index.js

```diff
@@ -5,11 +5,19 @@ import {RequestTracker} from "./util/request-tracking.js";
 export type {Cache, CacheEntry, Result, IRequestHandler} from "./util/types.js";
 
 export type ResponseCache = $ReadOnly<Cache>;
+
 export const initializeCache = (source: ResponseCache): void =>
     ResCache.Default.initialize(source);
+
 export const fulfillAllDataRequests = (): Promise<ResponseCache> =>
     RequestTracker.Default.fulfillTrackedRequests();
 
+/**
+ * TODO(somewhatabstract): Export each cache type we implement.
+ *
+ * Is there a base type we export, like we do for RequestHandler?
+ */
```

Comment:
I assume this is handled in a future PR.

Reply:
Yes